server/pension/models.py

```diff
@@ -21,13 +21,13 @@ class InstrumentType(object):
     TYPE2 = 2
 
     choices = (
-        (TYPE1, _('')),
-        (TYPE2, _('')),
+        (TYPE1, _('ניירות ערך סחירים')),
```

Comment:
@ordavidil @Lemore "ניירות ערך סחירים" is "Marketable Security" (or "Securities" in plurals). There is another option of "ניירות ערך לא סחירים" and that's "Non-Marketable Security".
Those are under the column "instrument_type".